Stack/Opc.Ua.Bindings.WebSockets/Opc.Ua.Bindings.WebSockets.csproj

```diff
@@ -0,0 +1,60 @@
+<Project Sdk="Microsoft.NET.Sdk">
+ 
+  <PropertyGroup>
+    <TargetFrameworks>$(LibTargetFrameworks)</TargetFrameworks>
+    <AssemblyName>Opc.Ua.Bindings.WebSockets</AssemblyName>
+    <PackageId>OPCFoundation.NetStandard.Opc.Ua.Bindings.WebSockets</PackageId>
+    <RootNamespace>Opc.Ua.Bindings</RootNamespace>
+    <Description>OPC UA WebSockets Binding Library</Description>
+    <IsPackable>true</IsPackable>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <LangVersion>8.0</LangVersion>
+  </PropertyGroup>
+  
+  <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
+    <PackageId>$(PackageId).Debug</PackageId>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(SignAssembly)' == 'true'">
+    <DefineConstants>$(DefineConstants);SIGNASSEMBLY</DefineConstants>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net462|AnyCPU'">
+    <OutputPath>C:\Repositories\AC-LIC-Prototype - Sequencing - WebSockets\AC.LIC.Sequencing\bin\Debug\net5.0\</OutputPath>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net462'">
+    <Reference Include="System.Net.Http" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel" Version="2.1.3" />
```

Comment:
this kestrel version is end of life -- check the https project how to use latest kestrel...

Reply:
Done. I just remove all Microsoft.AspNetCore.Server.Kestrel dependecies and add the new Microsoft.AspNetCore.Http dependency.
```
    <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.2.2" />
```

No code changes were required at all, the test results remain unchanged (when execting each test on its own):
![image](https://user-images.githubusercontent.com/72544898/171811299-a5079cd0-3b88-4cfd-8fe2-10cda671d72a.png)